src/core/core.adapters.js

```diff
@@ -27,8 +29,14 @@ function abstract() {
  * @name Unit
  */
 
-/** @lends Chart._adapters._date */
-module.exports._date = {
+/**
+ * @class
+ */
+function DateAdapter(options) {
+	this.options = options;
```

Comment:
`this.options = options || {};`, so we don't need the extra checks in the time scale `update()` method.

Reply:
done